VictoriaAlbert/VictoriaAlbert/Photo.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  Photo.swift
+//  VictoriaAlbert
+//
+//  Created by Amber Spadafora on 11/10/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import Foundation
+
+class Photo {
+
+    var photoID: String
+    var object: String // example: "object": "Gimmel ring"
+    var date: String
+    var place: String
+    var title: String
+    
+    init(photoID: String, object: String, date: String, place: String, title: String) {
+        
+        self.photoID = photoID
+        self.object = object
+        self.date = date
+        self.place = place
+        self.title = title
+    }
+    
+    convenience init?(from dictionary: [String: Any]) {
+        if let object = dictionary["object"] as? String,
+            let date = dictionary["date_text"] as? String,
+            let place = dictionary["place"] as? String,
+            let title = dictionary["title"] as? String,
+            let photoID = dictionary["primary_image_id"] as? String {
+                self.init(photoID: photoID, object: object, date: date, place: place, title: title)
+        } else {
+            return nil
+        }
+        
+    }
+    
+    func createSmallImageUrl(photoID: String) -> String {
+        let firstSixChars: String = String(photoID.characters.prefix(6))
+        
```

Comment:
Be consistent with spacing. Try not to have more than one blank line anywhere, two at most.